FirebaseSegmentation/Tests/Sample/Podfile

```diff
@@ -0,0 +1,17 @@
+# Uncomment the next two lines for pre-release testing on internal repo
+#source 'sso://cpdc-internal/firebase'
+#source 'https://cdn.cocoapods.org/'
+
+source 'https://cdn.cocoapods.org/'
+
+target 'SegmentationSampleApp' do
+  # Comment the next line if you don't want to use dynamic frameworks
+  use_frameworks!
+  inherit! :search_paths
+  platform :ios, '8.0'
+
+  # Pods for SegmentationSampleApp
+  pod 'FirebaseCore'
```

Comment:
Use local version of FirebaseCore as well

Reply:
Done.